e2e/migrations/migrations_test.go

```diff
@@ -0,0 +1,176 @@
+package e2e
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func isSuccess(execCmd *exec.Cmd, retries int, keyword string) (bool, string) {
+	numAttempts := retries
```

Comment:
May want to reuse this: https://github.com/hashicorp/nomad/blob/master/testutil/wait.go#L20